### ISSUE TITLE
EfficientNet_b1 ACC is always 0

### ISSUE THREAD

**Issue by ReverseSystem001** [cavalleria/cavaface.pytorch#29]
Hi, Do you use the pretrained model when train EfficientNet_b1. when I train it without pretrained model, the top1 and top5 ACC is always 0.

**Comment by cavalleria** [comment]
No. how many epochs did you train?

**Comment by ReverseSystem001** [comment]
about 3 or 4. but my batchsize is 128. how many epoches did you train when the acc is not 0?

**Comment by cavalleria** [comment]
just after 1 epoch

**Comment by ReverseSystem001** [comment]
the arcface loss parameters s and m are default？may be my batch size is too small? 892379162 邮箱：892379162@qq.com 签名由 网易邮箱大师 定制 On 07/22/2020 15:33, cavalleria wrote: just after 1 epoch — You are receiving this because you authored the thread. Reply to this email directly, view it on GitHub, or unsubscribe.

**Comment by Tamminhdiep97** [comment]
Hi, can I ask about config when training EfficientNet? what learning rate did you use? did you use decay learning rate like in the paper of EfficientNet?

**Comment by cavalleria** [comment]
initial lr is 0.1, and use cosinelrdecay scheduler.

**Comment by Tamminhdiep97** [comment]
> initial lr is 0.1, and use cosinelrdecay scheduler.

Thank you for your super fast reply @@. You're awsome
